package-lock.json

```diff
@@ -174,89 +174,96 @@
     },
     "@types/babel-types": {
       "version": "7.0.7",
-      "resolved": "https://registry.npmjs.org/@types/babel-types/-/babel-types-7.0.7.tgz",
-      "integrity": "sha512-dBtBbrc+qTHy1WdfHYjBwRln4+LWqASWakLHsWHR2NWHIFkv4W3O070IGoGLEBrJBvct3r0L1BUPuvURi7kYUQ=="
+      "resolved": "https://a0us.jfrog.io/a0us/api/npm/npm/@types/babel-types/-/babel-types-7.0.7.tgz",
```

Comment:
You need to change your npmrc when updating the dependencies to avoid these references leaking into here.